numcodecs/blosc.pyx

```diff
@@ -393,6 +394,100 @@ def decompress(source, dest=None):
     return dest
 
 
+def decompress_partial(source, start, nitems, typesize=0, dest=None):
```

Comment:
Just for clarity, we can drop `typesize` and rely on the header. The `dtype` will already passed through to Blosc correctly from Zarr.